rlw.sh

```diff
@@ -130,6 +130,7 @@ roblox-install () {
 
 playerwrapper () {
 	printf '%b\n' " > begin playerwrapper ()\n---"
+	spawndialog warning "Make sure you've logged in to either Studio or Firefox.\nWithout being logged in, this will make you join as a guest."
```

Comment:
We have enough popup dialogs already... Can you at least add some check so this doesn't show up every time?